install/machineapioperator_02_deployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: machine-api-operator
+  namespace: kube-system
+  labels:
+    k8s-app: machine-api-operator
+    managed-by-channel-operator: "true"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: machine-api-operator
+  template:
+    metadata:
+      labels:
+        k8s-app: machine-api-operator
+        tectonic-app-version-name: machine-api
+    spec:
+      containers:
+      - name: machine-api-operator
+        image: quay.io/coreos/machine-api-operator:b6a04c2 
```

Comment:
this will need to point here now https://hub.docker.com/r/openshift/origin-machine-api-operator/tags/

Reply:
kept it same as what is used today to keep new changes at minimum. Thought once we have new changes working with same old image, image update can be next step.